content/applications/hr/recruitment/team_performance.rst

```diff
@@ -0,0 +1,94 @@
+==========================
+Team performance reporting
+==========================
+
+The *Team Performance* report in the **Recruitment** app shows how many applicants each recruiter is
+managing.
+
+The information is determined by the person that populates the :ref:`recruiter field
+<recruitment/applicant-details>` on each applicant form.
+
+Open report
+===========
+
+To access the *Team Performance* report, navigate to :menuselection:`Recruitment app --> Reporting
+--> Team Performance`.
+
+This number of :guilabel:`In Progress`, :guilabel:`Hired`, and :guilabel:`Refused` applicants for
+each recruiter is displayed in a default :icon:`fa-area-chart` :guilabel:`(Graph)` view. The
+information shown is for the :icon:`fa-filter` :guilabel:`Last 365 Days Applicant`.
+
+Hover the cursor over any column to view a popover window displaying the specific details for that
+column.
+
+.. image:: team_performance/team-performance.png
+   :align: center
+   :alt: The default bar chart of the team performance report.
+
+Pivot table view
+----------------
+
+For a more detailed view of the information in the :guilabel:`Team Performance` report, click the
+:icon:`oi-view-pivot` :guilabel:`(Pivot)` icon in the top-right corner. This displays all the
+information in a pivot table.
+
+In this view, the job positions are displayed in the rows, while the columns display the total
+number of applicants. The applicants are further organized by :guilabel:`# Applicant` (in process),
+:guilabel:`# Hired`, and :guilabel:`# Refused`.
+
+The displayed information can be modified, if desired.
+
+In this example, there are 19 total applicants. Out of those 19, eight have been hired, and three
+refused.
+
+From the data presented, the :guilabel:`Experienced Developer` job position is the most successful.
+This job position has the highest number of total applicants, as well as the most hires. In
+addition, the :guilabel:`Experienced Developer` has the least amount of refused applicants.
+
+This pivot table also shows that the :guilabel:`Chief Executive Officer` position is the hardest to
+fill, as it has the fewest total applicants.
+
+.. image:: team_performance/team-perf-pivot.png
+   :align: center
+   :alt: The detailed pivot table view.
```

Comment:
This might look good in an example admonition, but I can also see leaving it out so it doesn't segment it from the following use case. Just something to consider.

Reply:
In the reporting docs I've done, I have made a use case in each of them to show what the report can do, and I think it was suggested to have it as a section instead of an admonition, so that's why I took it out of an example block. But I appreciate the suggestion!